content/_index.md

```diff
@@ -28,7 +28,7 @@ Here you will learn to:
 
 #### What you will learn
 
-In approximately 2 working days (~13 working hours) you can achieve the following:
+In approximately 2 working days (~8 working hours) you can achieve the following:
```

Comment:
Isn't 8 hours 1 working day?

I wondering if we leave the gantt with some estimates in it, but then change this to "In a couple of days..."

Reply:
Good catch! It's easier if I just shorten it to `In approximately 8 working hours`. Then the two will be aligned.

Reply:
That works for me!